dev-guide/multi-operator-manager.md

```diff
@@ -0,0 +1,186 @@
+# MultiOperatorManager
+
+## Summary
+
+HCP and standalone topologies face a problem of inconsistency and duplicate effort to introduce features
+because they are two entirely distinct mechanisms for managing the same operands.
+While some differences are absolutely required (deployments vs static pods), many are simply capricous.
+This is about finding a way to safely refactor ourselves into a single path for managing these operands.
+
+## Motivation
+
+The duplicate effort, delays caused by that duplicate effort, and fragmentation of the ecosystem as
+OCP supports different features in same version of different form factors is painful for customers,
+platform extension authors, and developers.
+Additionally, it is impractical to expect HCP to be an expert in managing (install, configure, and update)
+every operand on the management cluster.
+Finding way to keep operand teams aligned to how the operand is managed in every form factor is beneficial,
+finding a way to do this without simply requiring more effort is even better, and providing a mechanism to 
+improve testability at the same time is better still.
+This enhancement aims at improving all of those.
+
+### Goals
+
+* Keep teams aligned to units of function across all form-factors (desired by everyone I think)
+* Dramatically reduce overhead of running operators (single-node and HCP constraints)
+* Dramatically reduce the delta between standalone and hypershift (all of us feel this pain)
+* Dramatically improve testability of operators (all of us feel this pain)
+
+### Non-Goals
+
+* Reimplement operators in some meaningful way. We're shooting for a succesive refactor.
+* Change any customer facing API
+
+## Proposal
+
+### Definition of Terms
+Before we get into the changes, we need to define some terms.
+
+1. configuration cluster - this is a theoretical (doesn't really exist) kuberentes cluster that contains
+   the configuration that the user provides and the operators interact with.
```

Comment:
Can we disambiguate "the user" by using specific Personas? e.g. is this is user a cluster admin, a service consumer, service provider, a developer, ...?